packages/odf/components/odf-dashboard/dashboard.tsx

```diff
@@ -67,56 +72,44 @@ export const ODFDashboard: React.FC = () => {
 const ODFDashboardPage: React.FC<ODFDashboardPageProps> = (props) => {
   const { t } = useCustomTranslation();
   const title = t('Data Foundation');
-  const [pages, setPages] = React.useState<DashboardTabProps[]>([
-    {
-      id: 'overview',
-      href: '',
-      name: t('Overview'),
-      component: ODFDashboard,
-      contextId: CONTEXT_ID,
-    },
-    {
-      id: 'systems',
-      href: 'systems',
-      name: t('Storage Systems'),
-      component: StorageSystemListPage,
-      contextId: CONTEXT_ID,
-    },
-  ]);
+  const staticPages: DashboardTabProps[] = React.useMemo(
+    () => [
+      {
+        id: 'overview',
+        href: '',
+        name: t('Overview'),
+        component: ODFDashboard,
+        contextId: ODF_DASHBOARD_CONTEXT,
+      },
+      {
+        id: 'systems',
+        href: 'systems',
+        name: t('Storage Systems'),
+        component: StorageSystemListPage,
+        contextId: ODF_DASHBOARD_CONTEXT,
+      },
+    ],
+    [t]
+  );
 
-  const isTab = React.useMemo(() => isHorizontalNavTab(CONTEXT_ID), []);
+  const isDashboardTab = React.useMemo(
+    () =>
+      compose(isHorizontalNavTab, (e: Extension) => {
+        if (e.properties.contextId === ODF_DASHBOARD_CONTEXT) return e;
+      }),
+    []
+  );
```

Comment:
```suggestion
  const isDashboardTab = (e) =>
     isHorizontalNavTab(e) && (e.properties.contextId === ODF_DASHBOARD_CONTEXT)
  );
```
This can prolly live outside the component safely.

Reply:
make sense... not sure why I used a hook !

Reply:
missing